src/core/components/index.js

```diff
@@ -0,0 +1,18 @@
+exports.goOnline = require('./go-online')
```

Comment:
Yeah, that way we have to worry about those commas, the way it is right now doesn't